acceptance/configure_director_test.go

```diff
@@ -543,5 +548,43 @@ properties-configuration:
 			"elb_names": ["my-elb"]
 	  }`))
 		})
+
+		It("configures the BOSH director using the API", func() {
```

Comment:
What is unique about this test? You are specifically testing that this returns a 207, but I don't get this at all from your description

Reply:
Agreed, that's an oversight on my part.